src/index.ts

```diff
@@ -187,12 +187,9 @@ export function declutter(arg: any): any {
   }
   let ret = { ...arg }
   // Marqeta can send back several empty fields on all responses... clean it
-  if (isEmpty(ret.nextUrl)) {
+  if (isEmpty(ret.metadata)) {
```

Comment:
What is in the `metadata`? You say we can't set it in the `update()` function... and here, you're dropping it before anyone can see it... so what's it used for?

The docs say:

> Associates any additional metadata you provide with the business.

which makee me think they have to allow us to set it - both in the `create()` and `update()`... are you really sure we can't?

And if we can, then we _certainly_ don't want to delete it here - Right?

Reply:
Ss noted in slack, all one can do with the `metadata` property is `add` and `update`.  Removing a property is not allowed. Of course, a metadata property can be set to `empty`.